src/components/App.js

```diff
@@ -140,7 +149,7 @@ class App extends Component {
             <li>
               <button
                 onClick={() => {
-                  this.props.createSigningJob(this.props.files);
+                  this.props.createSigningJob(this.props.files, 'ios');
```

Comment:
enable android when it's ready

Reply:
This is fixed in a PR that I'll be submitting later today.